mlir/include/mlir/Interfaces/CallInterfaces.td

```diff
@@ -17,6 +17,53 @@
 
 include "mlir/IR/OpBase.td"
 
+
+/// Interface for operations with arguments attributes (both call-like
+/// and callable operations).
+def ArgumentAttributesOpInterface : OpInterface<"ArgumentAttributesOpInterface"> {
```

Comment:
I mentioned this in the RFC, but I'd avoid the additional interface all together. I don't think there is a lot of benefit added here over the cost on the generated code. Can this just be a list of additional methods that get added to the interface methods in CallOpInterface and CallableOpInterface?